Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:20.04
+ARG VERSION=1.0.3
+ENV VERSION=${VERSION}
+RUN apt-get update && apt-get install -y git wget
+RUN wget https://github.com/crate-ci/typos/releases/download/v${VERSION}/typos-v${VERSION}-x86_64-unknown-linux-gnu.tar.gz && \
+    tar -xzvf typos-v${VERSION}-x86_64-unknown-linux-gnu.tar.gz && \
+    mv typos /usr/local/bin
+COPY entrypoint.sh /entrypoint.sh
+WORKDIR /github/workspace
+ENTRYPOINT ["/bin/bash", "/entrypoint.sh"]
```

Comment:
That's a good point! I can a base typos container to the root, and for now a (mostly duplicate) version for GitHub actions in a docker subfolder. Then if you want to provide a container base with typos, it would b possible to set up an automated build and deploy for the container in the root, and have the actions use it as a base.

Reply:
We can also delete the Dockerfile from the root now, and keep the actions content hidden in the docker/ subfolder. If you decide to add a container for typos with automated builds you could add it back. Let me know your preference!

Reply:
I can go either way but if we are getting rid of the `extend` config, I suspect we can get rid of the entry point.

Reply:
I don't follow here - I'll push changes in a little bit and we can discuss what you'd like.

Reply:
Its moved, its ok.  We can always iterate on this later ayways